### PR TITLE
general code clean

### PR DESCRIPTION
**Pull request by auvipy**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 language: python
 cache: pip
-dist: xenial
+dist: bionic
 sudo: false
 
 python:
-  - 3.5
+  - 3.8
```
Comment (OskarPersson) [inline .travis.yml]: 3.8 is already listed
```suggestion
```

File: rest_framework_extensions/key_constructor/bits.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 from django.db.models.query import EmptyQuerySet
 from django.db.models.sql.datastructures import EmptyResultSet
 
-from django.utils.encoding import force_text
+from django.utils.encoding import force_str
```
Comment (OskarPersson) [inline rest_framework_extensions/key_constructor/bits.py]: Requires update on line 219

Comment (auvipy) [reply rest_framework_extensions/key_constructor/bits.py]: can u push on my branch?

Comment (OskarPersson) [reply rest_framework_extensions/key_constructor/bits.py]: Nope

Comment (OskarPersson) [reply rest_framework_extensions/key_constructor/bits.py]: You will have to add me as member then

**Comment by auvipy** [comment]
you can send a pr against 38 branch :)

**Comment by OskarPersson** [comment]
See #285 and #286

**Pull request merged**